ziti/cmd/verify/ops_verify_network.go

```diff
@@ -0,0 +1,263 @@
+/*
+	Copyright NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+package verify
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/openziti/ziti/common"
+)
+
+type network struct {
+	controller
+	controllerConfig string
+	routerConfig     string
+	verbose bool
+}
+
+func NewVerifyNetwork() *cobra.Command {
+	n := &network{}
+
+	cmd := &cobra.Command{
+		Use:   "verify-network",
```

Comment:
Should probably be `verify-network -c <controller-config-file> -r <router-config-file> [--verbose]`

Reply:
I'm torn. it's helpful, but it also feels easy to screw up when changing/adding flags. Also seems like with the flags enumerated on help it's less important. happy to discuss